src/tilde_timing_monitor/README.md

```diff
@@ -101,10 +124,10 @@ cp ~/colcon_ws/tilde_lite/src/tilde_timing_monitor/config/tilde_path_info.yaml .
 ros2 launch tilde_timing_monitor tilde_timing_monitor_node.launch.xml config_file:=tilde_path_info.yaml mode:=test
 ```
 
-| param      | value          | content                       | default |
-| ---------- | -------------- | ----------------------------- | ------- |
-| mode       | by config file | Measurement path type         | test    |
-| statistics | bool           | statistics collection control | true    |
+| param      | value          | content               | default |
+| ---------- | -------------- | --------------------- | ------- |
+| mode       | by config file | Measurement path type | test    |
+| debug_ctrl | bool           | debug control         | true    |
```

Comment:
```suggestion
| debug | bool           | debug control         | true    |
```

Reply:
I'm not sure but it looks `debug_ctrl` is used in the code for debug purpose but it is named `debug` as for external parameter.

Reply:
I'm not sure but it looks `debug_ctrl` is used in the code for debug purpose but it is named `debug` as for external parameter.

Reply:
As you said, "debug" is correct.